docker-run-opencv-ovms.sh

```diff
@@ -199,6 +199,7 @@ sleep 5
 PIPELINE_PROFILE="${PIPELINE_PROFILE:=grpc_python}"
 echo "starting client(s) with pipeline profile: $PIPELINE_PROFILE ..."
 docker run --network host $cameras $TARGET_USB_DEVICE $TARGET_GPU_DEVICE --user root --privileged --ipc=host --name $CLIENT_CONTAINER_NAME \
+-e MQTT="$MQTT" \
```

Comment:
we should set MQTT env to some default value like `127.0.0.1` or `localhost`  if $MQTT is empty or not set.

Reply:
the reason I didn't do that is because if is not set, then there is no need to transform or try to connect to MQTT. This is for benchmarking purposes to avoid the extra overhead

Reply:
i see, ok